data/systemd/snapd.service

```diff
@@ -6,6 +6,7 @@ Requires=snapd.socket
 ExecStart=/usr/lib/snapd/snapd
 EnvironmentFile=/etc/environment
 Restart=always
+WatchdogSec=5m
```

Comment:
The watchdog should be poked at twice the frequency specifies here from what I recall. How do we ensure that?